warp-tls/ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## 3.3.4
+
+* Add `setAccept` for hooking the socket `accept` call.
```

Comment:
This could get a better description of what happened in the `warp-tls` package. Something like:

"Integrated customizable `accept` hook from `Network.Wai.Handler.Warp.Settings` (cf. `setAccept`)"

Reply:
Nice catch!  It absolutely should be bumped to `3.3.24`.  This and the changelog wording have been committed.